### PR TITLE
update ags_session_load

### PR DESCRIPTION
**Pull request by duchaoyu**
- update AGS_session_load
fix force pipe scale in FormDiagram
choose location of ForceDiagram
scale ForceDiagram

### CODE REVIEW

File: ui/Rhino/AGS/dev/AGS_session_load_cmd.py [hunk 1]
```diff
@@ -51,15 +55,32 @@ def RunCommand(is_interactive):
...
+            force_id = scene.add(force, name='Force', layer='AGS::ForceDiagram')
+            force = scene.find(force_id)
+
+            # choose location of ForceDiagram
+            force.artist.anchor_vertex = 0
+            force.artist.anchor_point = compas_rhino.rs.GetPoint("Set Force Diagram Location")
+
+            # calculate the scale factor for ForceDiagram
+            scale_factor = calculate_drawingscale(form.diagram, force.diagram)
+            print("scale factor of the ForceDiagram is %s" % scale_factor)
```
Comment (brgcode) [inline ui/Rhino/AGS/dev/AGS_session_load_cmd.py:80]: i would take the print statement out

File: ui/Rhino/AGS/dev/AGS_session_load_cmd.py [hunk 2]
```diff
@@ -51,15 +55,32 @@ def RunCommand(is_interactive):
 
         if 'form' in data and data['form']:
             form = FormDiagram.from_data(data['form'])
-            scene.add(form, name='Form', layer='AGS::FormDiagram')
+            form_id = scene.add(form, name='Form', layer='AGS::FormDiagram')
+            form = scene.find(form_id)
+
+            # calculate the scale factor for FormDiagram
+            drawingscale_forces = calculate_drawingscale_forces(form.diagram)
+            form.artist.settings['scale.forces'] = drawingscale_forces
```
Comment (ricardoavelino) [inline ui/Rhino/AGS/dev/AGS_session_load_cmd.py:63]: What if we  first search from the `drawing_scale_forces` and `scale_factor` already stored in the session? They should be stored within `form.artist.settings['scale.forces']` and `force.artist.scale` already. Also, when we load the force, there should already have some information about the position of the `anchor_point`. I think it is better try recovering these information before and then just ask for user selection/scale calculation if we don't find it...

Comment (brgcode) [reply ui/Rhino/AGS/dev/AGS_session_load_cmd.py:63]: true. but i don't think any of that information is currently stored in the session. and the rhino-objects and the artist have no serialisation mechanism yet. working on that though... let's leave it like this for now and update as soon as the serialisation mechanisms are in place...

Comment (duchaoyu) [reply ui/Rhino/AGS/dev/AGS_session_load_cmd.py:63]: Same as Tom said... these attributes are saved in the artist not in the diagram.... that's why they need to be reculated and the anchor needs to be chosen again.

Comment (ricardoavelino) [reply ui/Rhino/AGS/dev/AGS_session_load_cmd.py:63]: Ok!

**Pull request merged**